lib/MailQueueHandler.php

```diff
@@ -325,6 +325,11 @@ protected function sendEmailToUser($userName, $email, $lang, $timezone, $maxTime
 			return true;
 		}
 
+		if (!$this->mailer->validateMailAddress($email)) {
+			$this->logger->warning('Notification for user "{user}" not sent because the email address "{email}" is invalid.', ['user' => $userName, 'email' => $email]);
+			return true;
```

Comment:
should we really delete them? or maybe just wait and send them once it's fixed?

Reply:
Don't know :) Usually the addresses are validated. Mind to ask the ldap / saml experts? It's more likely to happen with such setups.

Reply:
I choosed return true because a invalid email address is nothing temporary. Without manual action it will still fail with the next run.

Reply:
Well it could then send the emails once the admin fixed it. But welli don't care too much